### PR TITLE
API-1835: manifestclient/write_roundtripper: refine the the controller name annotation key

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: pkg/manifestclient/write_roundtripper.go [hunk 1]
```diff
@@ -152,7 +152,7 @@ func (mrt *writeTrackingRoundTripper) roundTrip(req *http.Request) ([]byte, erro
 		setAnnotationFor(bodyObj.(*unstructured.Unstructured), DeletionNameAnnotation, metadataName)
 	}
 	if controllerName := ControllerNameFromContext(req.Context()); len(controllerName) > 0 {
```
[the remark this answers is not in the document]
Comment (deads2k) [reply pkg/manifestclient/write_roundtripper.go:154]: lgtm otherwise.

[the remark this answers is not in the document]
Comment (p0lyn0mial) [reply pkg/manifestclient/write_roundtripper.go:154]: good idea, thx.

[the remark this answers is not in the document]
Comment (p0lyn0mial) [reply pkg/manifestclient/write_roundtripper.go]: done, ptal.

**Pull request merged**